### PR TITLE
Added E2E Tests for Session and Multichain modules

### PR DESCRIPTION
**Pull request by VGabriel45**
# Summary

Added E2E tests for Multichain Validation Module and Session Validation Module

Related Issue: #SMA-519

## Change Type

- [ ] Bug Fix
- [ ] Refactor
- [x] New Feature
- [ ] Breaking Change
- [ ] Documentation Update
- [ ] Performance Improvement
- [ ] Other

# Checklist

- [x] My code follows this project's style guidelines
- [x] I've reviewed my own code
- [ ] I've added comments for any hard-to-understand areas
- [ ] I've updated the documentation if necessary
- [x] My changes generate no new warnings
- [x] I've added tests that prove my fix is effective or my feature works
- [x] All unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published

# Additional Information

### CODE REVIEW

File: packages/account/tests/account.e2e.spec.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { PaymasterMode } from "@biconomy/paymaster";
+import { BiconomyPaymaster, PaymasterMode } from "@biconomy/paymaster";
 import { TestData } from ".";
 import { createSmartWalletClient } from "../src/index";
 import { Hex, encodeFunctionData, parseAbi } from "viem";
-import { UserOperationStruct } from "@alchemy/aa-core";
```
Comment (AmanRaj1608) [inline packages/account/tests/account.e2e.spec.ts:5]: Linting issue

File: .gitignore [hunk 2]
```diff
@@ -67,6 +67,9 @@ packages/transak/package-lock.json
 package-lock.json
 yarn.lock
 
+#session storage files
+packages/account/tests/modules/utils/sessionStorageData
```
Comment (joepegler) [inline .gitignore:71]: Reminder to fix this. Apologies

**Review by joepegler — APPROVED** [review]
LGTM, nice job

File: packages/account/tests/account.e2e.spec.ts (hunk 1 above)
Comment (VGabriel45) [reply packages/account/tests/account.e2e.spec.ts:5]: Linting is now passing

**Review by AmanRaj1608 — APPROVED** [review]

**Pull request merged**